### PR TITLE
KFSPTS-6354: re-skin PreEncumbrance GLPE tab

### PR DESCRIPTION
**Pull request by bhutchinson**

### CODE REVIEW

File: src/main/webapp/WEB-INF/tags/fp/preEncumbranceGeneralLedgerPendingEntries.tag [hunk 1]
```diff
@@ -69,65 +88,64 @@
 			<kul:htmlAttributeHeaderCell attributeEntry="${entryAttributes.financialDocumentReversalDate}" hideRequiredAsterisk="true" scope="col"/>
 		</tr>
 		<logic:iterate id="generalLedgerPendingEntry" name="KualiForm" property="${generalLedgerPendingEntriesProperty}" indexId="ctr">
-			<tr>
-				<th class="datacell center"><html:hidden property="${generalLedgerPendingEntryProperty}[${ctr}].transactionLedgerEntrySequenceNumber" write="true" value="${generalLedgerPendingEntry.transactionLedgerEntrySequenceNumber}" /></th>
-				<td class="datacell center">
+            <tr class="${ctr % 2 == 0 ? "highlight" : ""}">
```
Comment (jhulslander) [inline src/main/webapp/WEB-INF/tags/fp/preEncumbranceGeneralLedgerPendingEntries.tag]: I don't think it's a big deal, but I don't really like JSTL in line if statements like this.  They aren't as easy to read as breaking it out to a <c:choose> block.  I'll let you decide if it is worth chaning.

**Comment by jhulslander** [comment]
Changes look good, I had one small comment, I shall wait for feedback.

File: src/main/webapp/WEB-INF/tags/fp/preEncumbranceGeneralLedgerPendingEntries.tag (hunk 1 above)
Comment (bhutchinson) [inline src/main/webapp/WEB-INF/tags/fp/preEncumbranceGeneralLedgerPendingEntries.tag]: That's copied directly from the base code, so I would prefer to keep it as is.

Comment (jhulslander) [inline src/main/webapp/WEB-INF/tags/fp/preEncumbranceGeneralLedgerPendingEntries.tag]: fair enough

**Pull request merged**